kube-flagd-proxy/cmd/root.go

```diff
@@ -0,0 +1,75 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	cfgFile string
+	Version string
+	Commit  string
+	Date    string
+	Debug   bool
+)
+
+var rootCmd = &cobra.Command{
+	Use:               "flagd",
+	Short:             "Flagd-kube-proxy allows flagd to subscribe to CRD changes without the required permissions.",
+	Long:              ``,
+	DisableAutoGenTag: true,
+	// Uncomment the following line if your bare application
+	// has an action associated with it:
+	// Run: func(cmd *cobra.Command, args []string) { },
```

Comment:
These can be removed and kept clean

Reply:
sure, will remove these lines from the flags cmd as well